doc/Requirements Specification/Parts/Application.lyx

```diff
@@ -238,7 +238,7 @@ name "/A40/"
 
 \end_inset
 
- Beagle can be used for the verifying of design and implementation.
+ Beagle can be used for the verification of design and implementation.
```

Comment:
I’d prefer “Beagle can be used to verify design and implementation”. Simpler. What do you think?